keyboards/yushakobo/quick7/keymaps/tester/keymap.c

```diff
@@ -0,0 +1,157 @@
+/* Copyright 2020 yushakobo
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define FAVORITE_COLOR HSV_CYAN
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BASE
+};
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+    YUSHAURL = SAFE_RANGE,
+    KEY_00,
+    KEY_01,
+    KEY_02,
+    KEY_10,
+    KEY_11,
+    KEY_12,
+    KEY_20,
+    KEY_21,
+    KEY_22
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [_BASE] = LAYOUT(
+        KEY_00, KEY_01, KEY_02,
+        KEY_10, KEY_11, KEY_12,
+        KEY_20, KEY_21, KEY_22
+    )
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
```

Comment:
This looks like the "heatmap" animation for the rgb matrix.  

May be worth implementing that on the macro pad, instead.